src/app/components/status-indicator/index.html

```diff
@@ -0,0 +1,112 @@
+<stache
+  pageTitle="Status indicator"
+  windowTitle="Status indicator"
+  layout="container"
+	showTableOfContents="true">
+
+<p>
+	The status indicator classes create styled text to draw attention to data that needs to be addressed. A status is "a state or condition at a particular time". Statuses fall into four categories: success, warning, danger, and info. Examples include late, submitted, awaiting approval, etc.
+</p>
+
+<stache-page-anchor>
+	Status indicator classes
+</stache-page-anchor>
+
+  <div class="definition-table definition-table-wide">
+    <div class="definition-item">
+      <div class="sky-text-info">Info</div>
```

Comment:
When I serve this locally, there is no icon before "Info."

Reply:
It won't show up until that change is merged/released in SKY UX